### ISSUE TITLE
[Bug] [Unsorted] Requested Clipboard operation did not succeed

### ISSUE THREAD

**Issue by Alistair1231** [ChaosRecipeEnhancer/ChaosRecipeEnhancer#601]
# Prerequisites

Please answer the following questions for yourself before submitting an issue.

- [X] I am running the latest release (not a beta / pre-release)
- [X] I checked the FAQ and found no common resolution for my issue
- [X] I checked to make sure that this issue has not already been filed (by doing a quick search in the ![issue tracker](https://github.com/ChaosRecipeEnhancer/ChaosRecipeEnhancer/issues))

# Expected Behavior

Filter being updated.

# Current Behavior

What is the current behavior?

This error:
Reporting an exception I received while using Chaos Recipe Enhancer v3.23.1005:
```
The following exception occurred: Dispatcher.UnhandledExceptionSystem.Runtime.InteropServices.ExternalException (0x800401D0): Requested Clipboard operation did not succeed.
   at System.Windows.Forms.Clipboard.SetDataObject(Object data, Boolean copy, Int32 retryTimes, Int32 retryDelay)
   at ChaosRecipeEnhancer.UI.Services.FilterManipulation.ReloadFilterService.ReloadFilter() in D:\PoE Tools\ChaosRecipeEnhancer\EnhancePoEApp\src\App\ChaosRecipeEnhancer.UI\Services\FilterManipulation\ReloadFilterService.cs:line 53
   at ChaosRecipeEnhancer.UI.Windows.SetTrackerOverlayViewModel.RunReloadFilter() in D:\PoE Tools\ChaosRecipeEnhancer\EnhancePoEApp\src\App\ChaosRecipeEnhancer.UI\Windows\SetTrackerOverlayViewModel.cs:line 357
   at ChaosRecipeEnhancer.UI.Windows.SetTrackerOverlayWindow.RunReloadFilter() in D:\PoE Tools\ChaosRecipeEnhancer\EnhancePoEApp\src\App\ChaosRecipeEnhancer.UI\Windows\SetTrackerOverlayWindow.xaml.cs:line 138
```

## Steps to Reproduce

Please provide detailed steps for reproducing the issue.

1. Start app
2. Select Filter
3. Reload Filter
4. you get the error.

## Context

Please provide any relevant information about your setup. This is important in case the issue is not reproducible except for under certain conditions.

* Chaos Recipe Enhancer Version: v3.23.1005
* Operating System: Win 11 23H2
* Chaos Recipe Enhancer Settings (XML file or screenshot will suffice):

```xml
<?xml version="1.0" encoding="utf-8"?>
<configuration>
    <userSettings>
        <ChaosRecipeEnhancer.UI.Properties.Settings>
            <setting name="PathOfExileAccountName" serializeAs="String">
                <value>______</value>
            </setting>
            <setting name="PoEAccountConnectionStatus" serializeAs="String">
                <value>1</value>
            </setting>
            <setting name="LeagueName" serializeAs="String">
                <value>Necropolis</value>
            </setting>
            <setting name="StashTabIndices" serializeAs="String">
                <value>1</value>
            </setting>
            <setting name="AutoFetchOnRezoneEnabled" serializeAs="String">
                <value>True</value>
            </setting>
            <setting name="PathOfExileClientLogLocation" serializeAs="String">
                <value>C:\Program Files (x86)\Steam\steamapps\common\Path of Exile\logs\Client.txt</value>
            </setting>
            <setting name="SetTrackerOverlayDisplayMode" serializeAs="String">
                <value>2</value>
            </setting>
            <setting name="LootFilterManipulationEnabled" serializeAs="String">
                <value>True</value>
            </setting>
            <setting name="LootFilterFileLocation" serializeAs="String">
                <value>C:\Users\alist\Documents\My Games\Path of Exile\myDef.filter</value>
            </setting>
            <setting name="LootFilterIconsEnabled" serializeAs="String">
                <value>True</value>
            </setting>
            <setting name="SetTrackerOverlayTopPosition" serializeAs="String">
                <value>-17.6</value>
            </setting>
            <setting name="SetTrackerOverlayLeftPosition" serializeAs="String">
                <value>19.200000000000003</value>
            </setting>
            <setting name="UpgradeSettingsAfterUpdate" serializeAs="String">
                <value>False</value>
            </setting>
            <setting name="PathOfExileApiAuthTokenExpiration" serializeAs="String">
                <value>03/31/2024 22:53:13</value>
            </setting>
            <setting name="PathOfExileApiAuthToken" serializeAs="String">
                <value>__________________________________</value>
            </setting>
            <setting name="FetchStashHotkey" serializeAs="String">
                <value>F1</value>
            </setting>
            <setting name="ToggleStashTabOverlayHotkey" serializeAs="String">
                <value>F3</value>
            </setting>
            <setting name="SettingsWindowNavIndex" serializeAs="String">
                <value>3</value>
            </setting>
            <setting name="ReloadFilterHotkey" serializeAs="String">
                <value>F2</value>
            </setting>
        </ChaosRecipeEnhancer.UI.Properties.Settings>
    </userSettings>
</configuration>

```

## Failure Logs

Please include any relevant log snippets or files here.

**Comment by Alistair1231** [comment]
~~After changing to vanilla Filterblade it seems to work? If that changes, I will reopen.~~
I got confused since the Filter modification does work.
The only part that does not work for me is the reloading, and even that only sometimes.

**Comment by HiMarioLopez** [comment]
Howdy! I (re) released v3.24 today with included updates to the filter manipulation - including some stability updates and more graceful error handling. If you could do me a huge favor and give that version a try and let me know how it goes?

https://github.com/ChaosRecipeEnhancer/ChaosRecipeEnhancer/releases/tag/3.24.3000

As for the not reloading - make sure your offline filter file has a unique name that DOESN'T match any online filters you might have on your account. There can be 'name collisions' when we reload the filters

**Comment by Alistair1231** [comment]
Thanks for the tip. I will try the new version and make sure there are no name collisions. If the issue pops up again I will report back!
